packages/ubuntu_desktop_installer/lib/pages/keyboard_layout_page.dart

```diff
@@ -167,9 +181,8 @@ class _KeyboardLayoutPageState extends State<KeyboardLayoutPage> {
                 final client =
                     Provider.of<SubiquityClient>(context, listen: false);
                 final keyboard = KeyboardSetting(
-                    layout: _selectedLayoutName,
-                    variant: keyboardModel.layouts[_selectedLayoutIndex]
-                        .variants![_selectedVariantIndex].code!);
+                    layout: _selectedLayout!.code!,
+                    variant: _selectedVariant!.code!);
```

Comment:
- should the wizard action be disabled if there's no selected layout?
- KeyboardLayout.variants is nullable => can the selected variant be null at this point?

Reply:
Very good point. I disabled the button if there's no layout/variant selected. This ensures that when the action is triggered, the selected variant isn't null.